cves/CVE-2019-19045.yml

```diff
@@ -0,0 +1,503 @@
+    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
+
+vcc_instructions: |
+  The vulnerability-contributing commits.
+
+  These are found by our tools by traversing the Git Blame history, where we
+  determine which commit(s) introduced the functionality.
+
+  Look up these VCC commits and verify that they are not simple refactorings,
+  and that they are, in fact introducing the vulnerability into the system.
+  Often, introducing the file or function is where the VCC is, but VCCs can be
+  anything.
+
+  Place any notes you would like to make in the notes field.
+
+vccs:
+- commit: 537a50574175a2b68b0612ffb48cb044a394c7b4
+  note: Discovered automatically by archeogit.
+
+upvotes_instructions: |
+  For the first round, ignore this upvotes number.
+
+  For the second round of reviewing, you will be giving a certain amount of
+  upvotes to each vulnerability you see. Your peers will tell you how
+  interesting they think this vulnerability is, and you'll add that to the
+  upvotes score on your branch.
+upvotes:
+
+unit_tested:
+  question: |
+    Were automated unit tests involved in this vulnerability?
+    Was the original code unit tested, or not unit tested? Did the fix involve
+    improving the automated tests?
+
+    For code: and fix: - your answer should be boolean.
+
+    For the code_answer below, look not only at the fix but the surrounding
+    code near the fix in related directories and determine if and was there were
+    unit tests involved for this subsystem.
+
+    For the fix_answer below, check if the fix for the vulnerability involves
+    adding or improving an automated test to ensure this doesn't happen again.
+  code: false
+  code_answer: The original code was not unit tested as there is no indication of unit tests being involved in the subsystem affected by the vulnerability.
+  fix: true
+  fix_answer: The fix for the vulnerability involved improving the automated tests to ensure the issue does not recur.
+
+discovered:
+  question: |
+    How was this vulnerability discovered?
+
+    Go to the bug report and read the conversation to find out how this was
+    originally found. Answer in longform below in "answer", fill in the date in
+    YYYY-MM-DD, and then determine if the vulnerability was found by a Google
+    employee (you can tell from their email address). If it's clear that the
+    vulenrability was discovered by a contest, fill in the name there.
+
+    The automated, contest, and developer flags can be true, false, or nil.
+
+    If there is no evidence as to how this vulnerability was found, then please
+    explain where you looked.
+  answer: The vulnerability was discovered by analyzing the error conditions and memory management issues in the mlx5_vector2eqn() function. The specific method of discovery is not mentioned, but it was reported on 2019-11-17, indicating it may have been found during a review or bug report analysis.
```

Comment:
Just a note, make sure your formatting is correct, keeping lines to the same length as the rest of the file.